packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -75,19 +93,29 @@ const extractObjectExpression = (node: t.ObjectExpression, meta: Metadata): CSSO
 
       // Time to add this key+value to the CSS string we're building up.
       css += `${kebabCase(key)}: ${value};`;
-    } else if (t.isSpreadElement(prop) && t.isIdentifier(prop.argument)) {
+    } else if (t.isSpreadElement(prop)) {
       // We found a object spread such as: `...mixinIdentifier`.
-      const binding = meta.parentPath.scope.getBinding(prop.argument.name);
-      const resolvedBinding = resolveBindingNode(binding, meta);
+      const expression = tryGettingCalleeFromCallExpression(prop.argument);
+      let resolvedBinding = undefined;
 
-      if (!resolvedBinding) {
-        throw buildCodeFrameError('Variable could not be found', prop.argument, meta.parentPath);
+      if (t.isIdentifier(expression)) {
+        const binding = meta.parentPath.scope.getBinding(expression.name);
+        resolvedBinding = resolveBindingNode(binding, meta);
+
+        if (!resolvedBinding) {
+          throw buildCodeFrameError('Variable could not be found', prop.argument, meta.parentPath);
+        }
       }
 
-      if (t.isObjectExpression(resolvedBinding.node)) {
-        const result = extractObjectExpression(resolvedBinding.node, resolvedBinding.meta);
+      const { value: propValue, meta: updatedMeta } = traverseExpressionWithEvaluation(
+        expression,
+        meta
+      );
+
+      if (t.isObjectExpression(propValue)) {
+        const result = extractObjectExpression(propValue, updatedMeta);
 
-        if (resolvedBinding.source === 'import' && result.variables.length > 0) {
+        if (resolvedBinding && resolvedBinding.source === 'import' && result.variables.length > 0) {
```

Comment:
nit: i think we can remove `resolvedBinding` here

Reply:
Yeah. I can completely remove this check. I just kept it as it was earlier. Maybe I can just check for `ObjectExpression` and move on. What you say? Only problem I can see is it might break on runtime instead of compile time.

Reply:
let's leave it